sigmf/sigmffile.py

```diff
@@ -511,13 +547,33 @@ def validate(self):
         version = self.get_global_field(self.VERSION_KEY)
         validate.validate(self._metadata, self.get_schema())
 
-    def archive(self, name=None, fileobj=None):
+    def archive(self, file_path=None, fileobj=None, pretty=True):
```

Comment:
I agree that `file_path` is truly a better parameter name, but I'm not immediately convinced that it's worth breaking code for someone who provided an explicit parameter name in the method call ...